apps/client/src/app/modules/list/list-details-panel/list-details-panel.component.ts

```diff
@@ -95,13 +103,22 @@ export class ListDetailsPanelComponent implements OnChanges, OnInit {
 
   hasAlreadyBeenOpened: boolean;
 
+  server$ = this.authFacade.mainCharacter$.pipe(
+    map(char => char.Server)
+  )
+  private serializationHelper = new ListRowSerializationHelper(this.i18nTools, this.l12n, this.gt);
```

Comment:
Why are you using the helper like this instead of making it a service and let Angular handle dependencies for you (see any `.service.ts` file for example use of the dependency injection)?